polling_stations/apps/file_uploads/templates/file_uploads/file_detail.html

```diff
@@ -0,0 +1,46 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block page_title %}{{ object.filename }}{% endblock page_title %}
+
+{% block extra_page_css %}
+<style>
+  dt {
+      float: left;
+  }
+  dt::after{
+      content: ":";
+  }
+  dd {
+      margin-left: 25%;
+  }
+  dd::after {
+      content: " ";
+      display: block;
+      clear: left;
+  }
```

Comment:
another one for the "stuff that should be in the base theme" bucket
https://github.com/DemocracyClub/dc_base_theme/issues/49#issuecomment-574300635